src/hotspot/share/opto/type.cpp

```diff
@@ -62,12 +62,14 @@ const Type::TypeInfo Type::_type_info[Type::lastype] = {
   { Bad,             T_ARRAY,      "array:",        false, Node::NotAMachineReg, relocInfo::none          },  // Array
 
 #if defined(PPC64)
+  { Bad,             T_ILLEGAL,    "vectora:",      false, Op_VecA,              relocInfo::none          },  // VectorA.
```

Comment:
I think trailing "." in the comment is unnecessary. Please consider dropping before push.

Reply:
Hi Aleksey, I would not have put a dot there, but I just copied the line from the #else part which also has the dot.
So I'd rather leave it as-is.
Thanks for reviewing!

Reply:
Right. No biggie, ship it.